### ISSUE TITLE
? binary operator not implemented

### ISSUE THREAD

**Issue by bouk** [juliankrispel/decaf#65]
(not ternary!)

input:

```
a() ? ''
```

Error:

```
Error: "?" does not match field "operator": == | != | === | !== | < | <= | > | >= | << | >> | >>> | + | - | * | / | % | & | | | ^ | in | instanceof | .. of type BinaryExpression
```

Coffeescript output:

```
var ref;

if ((ref = a()) != null) {
  ref;
} else {
  '';
};
```

**Comment by juliankrispel** [comment]
Looking at that soon. Thanks for reporting

**Comment by juliankrispel** [comment]
fixed in #93
